src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -3881,6 +3881,57 @@ .maxstack  2
 }");
         }
 
+        [Fact, WorkItem(44902, "https://github.com/dotnet/roslyn/issues/44902")]
+        public void CopyCtor_NotInRecordType()
```

Comment:
>CopyCtor_NotInRecordType [](start = 20, length = 24)

What is this testing? It looks like the changes in the PR are specific to records.

Reply:
This is not related to the current PR, but I realized that I didn't test that scenario in previous PR. A constructor that looks like a copy constructor, but isn't in a record, isn't affected by change to skip initializer portion ([MethodCompiler](https://github.com/dotnet/roslyn/pull/45022/files#diff-339de703849fd37b6e84a4aafbf1d779R981)).